backend/typescript/rest/reviewRoutes.ts

```diff
@@ -20,10 +20,7 @@ reviewRouter.post("/", reviewRequestDtoValidator, async (req, res) => {
       body: req.body.body,
       byline: req.body.byline,
       featured: req.body.featured,
-      /*
-       *@TODO: uncomment when christine changes are merged
```

Comment:
Do you know what Christine's changes were? From the git blame looks like this has been here a while :thinking:

Reply:
it's that PR https://github.com/uwblueprint/ccbc/pull/3. At some point me, jessica and Christine were working on review service and user models at the same time we couldn't reference the user ID as a foreign key since the user table wasn't created yet.